### ISSUE TITLE
Schedule for "Sync users with Azure AD" wrong?

### ISSUE THREAD

**Issue by leonstr** [microsoft/o365-moodle#1456]
On install of **local_o365** the scheduled task **Sync users with Azure AD** (**\local_o365\task\usersync**) is scheduled to run every minute between 01:00 and 01:59. From **local/o365/db/tasks.php**:

```
    [
        'classname' => 'local_o365\task\usersync',
        'blocking' => 0,
        'minute' => '*',
        'hour' => '1',
        'day' => '*',
        'dayofweek' => '*',
        'month' => '*'
    ],
```

Presumably this is a mistake and it's supposed to run once every 24 hours?

**Comment by weilai-irl** [comment]
@leonstr, thank you for reporting this.

This is effectively the same as #1442. 
Work will be carried out in that issue.

**Comment by weilai-irl** [comment]
Hi @leonstr,

The bug has been fixed in the latest release (3.8.0.5 and 3.9.2) of the plugins. Thank you again for reporting the issue.

Regards,
Lai
